### DISCUSSION TITLE
User Role dont fix

### DISCUSSION THREAD

**Discussion by Thorzera** [opendcim/openDCIM#1511] [Q&A]
Hello everyone, I'm starting to use OpenDCIM and also improving my Linux skills, I wanted some help, after installing OpenDCIM, the default login user does not have any access role, it is blank, could anyone help me? if it is some configuration that I ended up not performing.

I'm using debian 12

![opendcim](https://github.com/opendcim/openDCIM/assets/147781260/ab1da156-56e1-43d8-992e-c46597913b6f)

**Comment by wilpig** [comment]
Follow the Ubuntu 22 install instructions.

https://github.com/opendcim/openDCIM/wiki/Ubuntu#ubuntu-2204-installation-instructions

**Reply by Thorzera** [reply]
I already redid the entire clean installation, but the problem still persists, I wanted to know if it was a specific cause.

**Reply by wilpig** [reply]
based on your picture and what you're describing you didn't follow it step by step or it would be working.  the most likely thing is that you opened the site prior to getting the authentication piece in place and it installed and didn't create the admin user correctly.  dump the db, and do that bit over and it should be fine.
